README.md

```diff
@@ -38,6 +38,15 @@ To download any of these models, please [follow this link](https://uts.nlm.nih.g
 - **Paper**: [What’s in a Summary? Laying the Groundwork for Advances in Hospital-Course Summarization](https://www.aclweb.org/anthology/2021.naacl-main.382.pdf)
 - ([more...](https://github.com/CogStack/MedCAT/blob/master/media/news.md))
 
+## Installation
+To install the latest version of MedCAT run the following command:
+```
+pip install medcat
+```
+To install the latest version of MedCAT without torch GPU support run the following command:
+```
+pip install medcat --extra_index_url https://download.pytorch.org/whl/cpu/
```

Comment:
lets have a sentence on why this might be useful to a potential user - i.e. saves a load of space on your venv / filesystem / docker container etc.

Reply:
@adam-sutton-1992  - can you address then I'll approve / merge - thanks